lib/spack/spack/cmd/module.py

```diff
@@ -10,6 +10,8 @@
 import spack.cmd.modules.dotkit
 import spack.cmd.modules.lmod
 import spack.cmd.modules.tcl
+import spack.cmd.modules.ups_version
```

Comment:
Once https://github.com/spack/spack/pull/11145 gets merged upstream I was planning on submitting a PR to use it here, removing the need for the top level command to know about every sub-command. See https://github.com/FNALssi/spack-dev/blob/master/dev/cmd/dev.py for an idea of how this might work.